test/framework/repository.py

```diff
@@ -129,27 +128,6 @@ def test_svnrepo(self):
         self.assertTrue(os.path.exists(os.path.join(repo.wc, 'trunk', 'README.md')))
         shutil.rmtree(repo.wc)
 
-    # this test is disabled because it fails in Travis as a result of bitbucket disabling TLS 1.0/1.1
-    # we can consider re-enabling it when moving to a more recent Ubuntu version in the Travis config
-    # (which implies dropping support for Python 2.6)
-    # cfr. https://github.com/easybuilders/easybuild-framework/pull/2678
-    def DISABLED_test_hgrepo(self):
-        """Test using HgRepository."""
-        # only run this test if pysvn Python module is available
-        try:
-            import hglib  # noqa
-        except ImportError:
-            print("(skipping HgRepository test)")
-            return
-
-        # GitHub also supports SVN
-        test_repo_url = 'https://kehoste@bitbucket.org/kehoste/testrepository'
-
-        repo = HgRepository(test_repo_url)
```

Comment:
Not testing this anymore means we should also deprecate the use of `HgRepository`, so we can drop support entirely in EasyBuild 5.0?

Reply:
We haven't tested this for ages, I just removed the dead code. No idea if you want to deprecate this as Mercury isn't exactly dead but I don't really care. If it works why not keep it. Might add a test later or deprecate it later.

PS: I love the comment above a mercury bitbucket URL about github and SVN ;)